radosgw_usage_exporter.py

```diff
@@ -83,7 +83,10 @@ def _setup_empty_prometheus_metrics(self):
     def _process_data(self, entry):
         # Recieves JSON object 'entity' that contains all the buckets relating
         # to a given RGW UID.
-        bucket_owner = entry['owner']
+        try:
```

Comment:
I think this is improperly to use try-except in this case.
Better to looking to entry to find properly key.

```py
        if 'owner' in entry:             
            bucket_owner = entry['owner']
        # Luminous                       
        elif 'user' in entry:            
            bucket_owner = entry['user'] 
```

Reply:
Make sense. will fix now